src/libs/ComposerUtils/index.native.js

```diff
@@ -1,6 +1,8 @@
 import lodashGet from 'lodash/get';
 import styles from '../../styles/styles';
 import updateIsFullComposerAvailable from './updateIsFullComposerAvailable';
+// eslint-disable-next-line import/extensions
```

Comment:
Why do we need this here? I only see it used once in another place?
```
desktop/dev.js
6-
7-// The `dist` folder is not part of the source and does not exist until we build or start Desktop
8:// eslint-disable-next-line import/extensions
9-require('./dist/main');
```

Reply:
The lint complains when we add an extension to the import.

If we remove the extension (.js), there will be a warning of dependency cycle (index.native.js -> index.native.js), so we need to explicitly add the extension.